packages/pigeon/lib/pigeon_lib.dart

```diff
@@ -282,6 +304,16 @@ class Pigeon {
       ));
     }
 
+    for (final ClassMirror enumMirror in enums) {
+      List<String> members = <String>[];
+      // The first 3 declarations and last two are trimmed as they are innate to
+      // enums in Dart and are not user defined values.
+      for (int i = 3; i < enumMirror.declarations.keys.length - 2; i++) {
```

Comment:
Do the ignored fields seem to have defined names?  Instead of filtering by index we could filter by the name of the fields which would be a bit more robust.

Reply:
Switched to check against a defined names set.